1209/A10C/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: Aminoacid Synthesizer
+owner: KBLES
```

Comment:
Capitalisation for the 'owner' field needs to match that which you used for the org directory.